Index: Team12-xcbc/src/main/java/com/example/team12xcbc/controller/test.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Team12-xcbc/src/main/java/com/example/team12xcbc/controller/test.java b/Team12-xcbc/src/main/java/com/example/team12xcbc/controller/Controller.java
rename from Team12-xcbc/src/main/java/com/example/team12xcbc/controller/test.java
rename to Team12-xcbc/src/main/java/com/example/team12xcbc/controller/Controller.java
--- a/Team12-xcbc/src/main/java/com/example/team12xcbc/controller/test.java	(revision d4d76e1e5bf34fd8683e87eeb23ffec46a654b2f)
+++ b/Team12-xcbc/src/main/java/com/example/team12xcbc/controller/Controller.java	(date 1668841374160)
@@ -1,4 +1,12 @@
 package com.example.team12xcbc.controller;
 
-public class test {
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/user")
+public class Controller {
+
 }
Index: Team12-xcbc/src/main/java/com/example/team12xcbc/Team12XcbcApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.team12xcbc;\r\n\r\nimport org.springframework.boot.SpringApplication;\r\nimport org.springframework.boot.autoconfigure.SpringBootApplication;\r\n\r\n@SpringBootApplication\r\npublic class Team12XcbcApplication {\r\n\r\n    public static void main(String[] args) {\r\n        SpringApplication.run(Team12XcbcApplication.class, args);\r\n    }\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Team12-xcbc/src/main/java/com/example/team12xcbc/Team12XcbcApplication.java b/Team12-xcbc/src/main/java/com/example/team12xcbc/Team12XcbcApplication.java
--- a/Team12-xcbc/src/main/java/com/example/team12xcbc/Team12XcbcApplication.java	(revision d4d76e1e5bf34fd8683e87eeb23ffec46a654b2f)
+++ b/Team12-xcbc/src/main/java/com/example/team12xcbc/Team12XcbcApplication.java	(date 1668840404847)
@@ -5,7 +5,6 @@
 
 @SpringBootApplication
 public class Team12XcbcApplication {
-
     public static void main(String[] args) {
         SpringApplication.run(Team12XcbcApplication.class, args);
     }
diff --git a/学号姓名/032002314黄霆楠.md b/学号姓名/032002314黄霆楠.md
deleted file mode 100644
